datacontract/imports/glue_importer.py

```diff
@@ -109,18 +109,17 @@ def get_glue_table_schema(database_name: str, table_name: str):
                     "Name": pk["Name"],
                     "Type": pk["Type"],
                     "Hive": True,
-                    "Comment": "Partition Key",
+                    "Comment": pk.get("Comment"),
```

Comment:
`"Comment": "Partition Key"` is overriding the column comment, I suggest to remove it and leave `required: true` as the only signal of the partition column

Reply:
ok

Reply:
@jochenchrist Thanks for the review! Is there anything else that needs to be done in this PR?